### PR TITLE
Mention that WebRTC stats should not leak whether mDNS resolution failed or succeeded

### PR DESCRIPTION
**Pull request by youennf**

### CODE REVIEW

**Comment by youennf** [comment]
Together with https://github.com/rtcweb-wg/mdns-ice-candidates/pull/124, this should fix https://github.com/rtcweb-wg/mdns-ice-candidates/issues/121.

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 1]
```diff
@@ -309,6 +316,13 @@ Statistics related to ICE candidates that are accessible to the web
 application MUST NOT contain the IP address of a local or remote mDNS
 candidate; the mDNS name SHOULD be used instead.
 
+Statistics SHOULD NOT leak whether the mDNS resolution succeeds or fails.
+For that reason, RTCIceCandidateStats objects as defined in {{WebRTCStats}}
+SHOULD be generated for any remote mDNS candidate submitted to the ICE agent,
+even if the mDNS candidate is ignored as part of {{processing}}.
+An implementation strategy to ignore candidates is to replace
```
Comment (tQsW) [inline draft-ietf-rtcweb-mdns-ice-candidates.md:323]:
```suggestion
An implementation strategy to obfuscate the address of an mDNS candidate in the statistics, regardless if it is resolved or not, is to replace
```

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 2]
```diff
@@ -309,6 +316,13 @@ Statistics related to ICE candidates that are accessible to the web
...
+the mDNS hostname of the ICE candidate with IP values "0.0.0.0"/"::".
```
Comment (tQsW) [inline draft-ietf-rtcweb-mdns-ice-candidates.md:324]: nit
```suggestion
the mDNS hostname of the ICE candidate with IP values "0.0.0.0" or "::".
```

**Review by tQsW — APPROVED** [review]

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 3]
```diff
@@ -309,6 +316,13 @@ Statistics related to ICE candidates that are accessible to the web
 application MUST NOT contain the IP address of a local or remote mDNS
 candidate; the mDNS name SHOULD be used instead.
 
+Statistics SHOULD NOT leak whether the mDNS resolution succeeds or fails.
+For that reason, RTCIceCandidateStats objects as defined in {{WebRTCStats}}
+SHOULD be generated for any remote mDNS candidate submitted to the ICE agent,
+even if the mDNS candidate is ignored as part of {{processing}}.
+An implementation strategy to obfuscate the address of an mDNS candidate in the statistics, regardless if it is resolved or not, is to replace
+the mDNS hostname of the ICE candidate with IP values "0.0.0.0" or "::".
```
Comment (juberti) [inline draft-ietf-rtcweb-mdns-ice-candidates.md:324]: Not sure I understand this guidance. Wouldn't we just want stats.address to contain the mDNS name?

**Pull request merged**